postgres/rdspostgres/rdspostgres.go

```diff
@@ -53,6 +55,40 @@ type Params struct {
 	TraceOpts []ocsql.TraceOption
 }
 
+type URLOpener struct {
+	// TraceOpts contains options for OpenCensus.
+	TraceOpts []ocsql.TraceOption
+}
+
+// Scheme is the URL scheme rdspostgres registers its URLOpener under on
+// postgres.DefaultMux.
+const Scheme = "rdspostgres"
+
+func init() {
+	postgres.DefaultURLMux().RegisterPostgres(Scheme, &URLOpener{})
+}
+
+func (uo *URLOpener) OpenPostgresURL(ctx context.Context, u *url.URL) (*sql.DB, error) {
+	cf := new(rds.CertFetcher)
+	for k, _ := range u.Query() {
+		// Only permit parameters that do not conflict with other behavior.
+		if k == "user" || k == "password" || k == "dbname" || k == "host" || k == "port" || k == "sslmode" || k == "sslcert" || k == "sslkey" || k == "sslrootcert" {
```

Comment:
As I discovered in #1560, this check makes it impossible to connect via Unix sockets. While I grant connecting over Unix sockets doesn't happen for RDS, I don't see these parameters as being overly harmful, so let's drop the check.